nixos/modules/misc/ids.nix

```diff
@@ -385,7 +385,7 @@
       virtuoso = 44;
       #rtkit = 45; # unused
       dovecot2 = 46;
-      #dovenull = 47; # unused
+      dovenull2 = 47;
```

Comment:
You can actually use `dovenull` instead.

Reply:
Would you prefer that? I used "dovenull2" because that is identical to the uid's name.

Reply:
Ah make sense. I have not spotted that.